README.md

```diff
@@ -18,12 +18,12 @@ limitations under the License.
 
 # Triton Model Analyzer
 
->**LATEST RELEASE:**<br>
-You are currently on the `main` branch which tracks
-under-development progress towards the next release. <br>The latest
-release of the Triton Model Analyzer is 1.32.0 and is available on
-branch
-[r23.11](https://github.com/triton-inference-server/model_analyzer/tree/r23.11).
+> [!Warning]
+> ##### LATEST RELEASE
+> You are currently on the `main` branch which tracks under-development progress towards the next release. <br>
+> The latest release of the Triton Model Analyzer is 1.35.0 and is available on branch
```

Comment:
1.32 -> 1.35? Why the big jump? Is this accurate?

Also, the link on line 25 is still pointing to 23.11

Reply:
I believe that happen due to mistake on review...  
Guess we can make it agnostic to the branch in future.